assets/src/edit-story/app/uploader/useUploader.js

```diff
@@ -29,13 +29,20 @@ import { __, sprintf } from '@wordpress/i18n';
  */
 import { useAPI } from '../../app/api';
 import { useConfig } from '../config';
-import { useMedia } from '../media';
 import createError from '../../utils/createError';
 
 function useUploader(refreshLibrary = true) {
-  const {
-    actions: { resetWithFetch },
-  } = useMedia();
+  // TODO(https://github.com/google/web-stories-wp/issues/1703):
```

Comment:
This feels like it needs to go into its own pull request.

Reply:
Done: https://github.com/google/web-stories-wp/pull/1799